modules/cnf-modifying-and-applying-the-default-profile.adoc

```diff
@@ -0,0 +1,15 @@
+// CNF-950 4.7 Modifying and applying the default profile
+// Module included in the following assemblies:
+//
+// *cnf-provisioning-and-deploying-a-distributed-unit.adoc
+
+[id="cnf-modifying-and-applying-the-default-profile_{context}"]
+= Modifying and applying the default profile
+
+Now you can apply the profile manually or with the toolset of your choice (for example, ArgoCD).
```

Comment:
s/Now/You can
Unless you want to state when "now" is, since this is in its own standalone module and that context is lost.

Reply:
your choice, such as ArgoCD. 

We try to avoid parentheticals when possible.

Reply:
Approved.